website/docs/docs/core/connect-data-platform/databricks-setup.md

```diff
@@ -48,17 +46,59 @@ pip is the easiest way to install the adapter:
+
+| Field     | Example | Description |
+| --------- | ------- | ----------- |
+|   `host`   | `yourorg.databrickshost.com` | The hostname of your cluster.<br/><br/>Don't include the `http://` or `https://` prefix.  |
+|   `http_path`   | `/sql/your/http/path`  | The http path to your SQL Warehouse or all-purpose cluster. |
+|  `schema`  | `my_schema`  | The name of a schema within your cluster's catalog. <br/><br/>It's _not recommended_ to use schema names that have upper case or mixed case letters.  |
+
+## Authentication parameters
+
+The `dbt-databricks` adapter supports both token-based authentication and OAuth client-based authentication. 
+
+| Field     | Example | Description |
+| --------- | ------- | ----------- |
+|  `token`  | `dapiXXXXXXXXXXXXXXXXXXXXXXX`  | The Personal Access Token (PAT) to connect to Databricks. This is required if you are using token-based authentication. |
+|  `client_id`  | `<oauth-client-id>`  | The client ID for your Databricks OAuth application. This is required if you are using OAuth-based authentication. |
+|  `client_secret`  | `XXXXXXXXXXXXXXXXXXXXXXXXXXX`  | The client secret for your Databricks OAuth application. This is required if you are using OAuth-based authentication. |
+|  `auth_type`  | `oauth`  | This is required if you are using OAuth-based authentication. You do not need to include this value if you are using token-based authentication. |
+
+
+## Additional parameters
+
+The following profile fields are optional to set up. They let you configure your cluster's session and dbt for your connection. 
+
+
+| Profile field                 | Example                          | Description                                                                                                 |
+| ----------------------------- | -------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| `threads`                     | `8`                              | How many threads dbt should use (default is `1`)                                                            |
+| `connect_retries`                     | `3`                              | How many times dbt should retry the connection to Databricks (default is `1`)                                                            |
+| `connect_timeout`                     | `1000`                              | How many seconds before the connection to Databricks should timeout (default behavior is no timeouts)                                                            |
+| `session_properties`          | `ansi_mode: true`         | Sets Databricks session properties used in the connection. Execute `SET -v` to see available options       |
 
-dbt-databricks can connect to the Databricks SQL Warehouses and all-purpose clusters. Databricks SQL Warehouses is the recommended way to get started with Databricks.
+#### Example profiles.yml for token-based authentication
```

Comment:
i'm wondering if this should be an h3 or h2 header? i also like the idea of using tabs so the user can switch between the two without much scrolling. if that makes sense, I'm happy to change this into tabs and share the preview.

Reply:
Not sure if this is what you're referring to or not, but I was kinda questioning the side nav hierarchy after I saw the deploy preview. I feel like these should maybe be represented on there in some capacity (even if it's one link to the section for `example profiles` - totally defer to your judgment though.

I was also kind of wondering if it'd be appropriate to nest the '[...] Parameters' sections _underneath_ the 'Configuring dbt-databricks' section. What do you think?

Reply:
yea great shout! i'm going to make some edits if you're ok with that? and will share the deploy